0003-interledger-protocol/0003-interledger-protocol.md

```diff
@@ -318,12 +318,18 @@ Relative errors indicate that the payment did not have enough of a margin in ter
 
 ### ILP Fulfillment Format
 
-Here is a summary of the fields in the ILP fulfillment format:
+This type of ILP packet is attached to a fulfillment and carries additional transport layer information that the sender may use to make further payments. Here is a summary of the fields in the ILP fulfillment format:
 
 | Field | Type | Short Description |
 |:--|:--|:--|
 | data | OCTET STRING | Transport layer fulfillment data |
 
+#### Example
+
+| length | data | extensions |
+|:--|:--|:--|:--|
+| 3 | 4 16 65 | 0 |
```

Comment:
You're still missing the packet type and the envelope length, and would be good to explain that `4 16 65` is the actual payload. Also, number of `:--|` should match number of columns in the markdown table. So more like:

#### Example

Consider a payload of `4 16 65` (payload length is `3`).
The length of the envelope content would then be `1+3+1 = 5`.
The ILP packet type is always `10` for fulfillment data, and extensions is always `0`, so on the wire
(and when passed between software modules like for instance ledger plugins),
the resulting ILP packet would look like this:

| type | content length | payload length | data | extensions |
|:--|:--|:--|:--|:--|
| 10 | 5 | 3 | 4 16 65 | 0 |

Reply:
Nice! I added this example. Thanks!